valet-key/Readme.md

```diff
@@ -4,47 +4,61 @@ This document describes the Valet Key Pattern example from the guide [Cloud Desi
 
-
 ## Running the Example
 
-You can run this example locally in the Visual Studio Windows Azure emulator. You can also run this example by deploying it to a Windows Azure Cloud Service.
+You can run this example locally in the Visual Studio Windows Azure emulator. You can also run this example by deploying it to a Windows Azure App Service.
 
-* Start Visual Studio using an account that has Administrator privileges ("Run as Administrator").
-* Open the solution you want to explore from the subfolders where you downloaded the examples.
-* Right-click on each role in Solution Explorer, select Properties, and ensure that the role is configured to generate diagnostic information.
+If you want to run this example locally
 
+* Start Visual Studio, set the ValetKet.Web project as startup
+* Edit the appsettings.json and provide values for these settings:
 
-* If you want to run the example on Windows Azure, provision a Windows Azure Cloud Service and deploy the application to it from Visual Studio. Alternatively you can run the example in the local Windows Azure emulator.
-
-* Start the Web Service and note the URL of the web role shown in the browser address bar.
-* Open the file app.config from the ValetKey.Client project and change the setting for serviceEndpointUrl to   [your-URL]**/api/values/**
-	* By default this is set to **http://127.0.0.1:81/api/values/**
+	1 - StorageConnectionString
+	2 - StorageKey
 
+	You should be able to get these values from the Azure Portal; go to the storage account, and from the
+	left menu, in the Security + networking section, select the option "Access Keys".
+	In the "Access Keys" dialog, click on "show keys"; you will be able to see and copy the key and the connection string.
 
+* Hit F5 to run the Web Api
 * Start a new instance of the ValetKey.Client project to upload the blob. Right-click the project, select Debug, and click Start new instance.
 
-* If you are running the application locally in the Windows Azure emulator:
-	* Open the Windows Azure Compute Emulator UI from the icon in the notification area.
-	* Select each role in turn and view the diagnostic information generated by Trace statements in the code.
-
-* If you are running the application on Windows Azure:
-	* Open the Server Explorer Window in Visual Studio and expand the Windows Azure entry.
-	* Expand Cloud Services and then expand the entry for the solution you deployed.
-	* Right-click each role instance and select View Diagnostics Data to see the diagnostic information generated by Trace statements in the code. This is written to the WADLogsTable in the Storage/Development/Tables section.
-
-
-
+* If you want to run the example on Windows Azure, provision a Windows Azure App Service and deploy the application to it from Visual Studio.
```

Comment:
Please rebrand.  Also this doesn't flow well.  If this is a whole other code path, you'll want to break out the local from remote running a bit clearer here.

Reply:
Done